lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingEvaluation.cxx

```diff
@@ -141,14 +141,58 @@ struct KrigingEvaluationPointFunctor
 
 }; // struct KrigingEvaluationPointFunctor
 
+struct KrigingEvaluationPointFunctor1D
+{
+  const Point & input_;
+  const KrigingEvaluation & evaluation_;
+  Scalar accumulator_;
+
+  KrigingEvaluationPointFunctor1D(const Point & input,
+                                  const KrigingEvaluation & evaluation)
```

Comment:
I dont see where that ctor is needed

Reply:
In https://github.com/openturns/openturns/pull/1429/files#diff-41b14220e03d778dfa73ce9f1e261d7eR186

Reply:
I meant the copy ctor(other, TBB::Split)